downstream/modules/platform/proc-gs-add-ee-to-job-template.adoc

```diff
@@ -13,10 +13,29 @@ Execution environments tied to an organization require Organization administrato
 
 .Procedure
 
-(ADD CONTENT)
+. From the navigation panel, select {MenuInfrastructureExecEnvironments}.
+. Click btn:[Create execution environment] to create an {ExecEnvShort}.
+. Enter the appropriate details into the following fields: 
+.. *Name* (required): Enter a name for the {ExecEnvShort}.
+.. *Image* (required): Enter the image name. The image name requires its full location (repository), the registry, image name, and version tag, as in the following example: 'quay.io/ansible/awx-ee:latestrepo/project/image-name:tag'
+.. Optional: *Pull*: Choose the type of pull when running jobs: 
+... *Always pull container before running*: Pulls the latest image file for the container.
+... *Only pull the image if not present before running*: Only pulls the latest image if none are specified.
+... *Never pull container before running*: Never pull the latest version of the container image.
++
+NOTE: If you do not set a type for pull, the value defaults to *Only pull the image if not present before running*.
```

Comment:
Use note admonition as follows:
[NOTE]
====
<text>
====

Reply:
Evidently this is a legit way in-line notes can be formatted: see [this reference](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntax). It does appear to be building correctly, so leaving as-is